extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -42,4 +42,40 @@ descendant_tag_list {
   tag: "A"
   tag: "SPAN"
   tag: "I-AMPHTML-SIZER"  # Only allowed when document is transformed AMP
+  tag: "IMG"
+}
+tags: {
+  html_format: AMP
+  tag_name: "IMG"
+  spec_name: "AMP-STORY-PLAYER IMG"
+  mandatory_ancestor: "AMP-STORY-PLAYER"
+  attrs: {
+    name: "loading"
+    value: "lazy"
+  }
+  attrs: { name: "data-amp-story-player-poster-img" }
+  attrs: { name: "alt" }
+  attrs: { name: "attribution" }
+  attrs: { name: "border" }  # Not valid html5 but commonly used and supported.
+  attrs: {
+    name: "decoding"
+    value: "async"
+    value: "auto"
+    value: "sync"
+  }
+  attrs: { name: "height" }  # Not valid html5 but commonly used and supported.
+  attrs: { name: "ismap" }
+  attrs: {
+    name: "longdesc"
+    value_url: {
+      protocol: "http"
+      protocol: "https"
+    }
+    disallowed_value_regex: "__amp_source_origin"
+  }
+  attrs: { name: "importance" }  # Not yet part of the html spec
+  attrs: { name: "intrinsicsize" }  # Not yet part of the html spec
```

Comment:
It these aren't part of the spec yet, maybe consider leaving them off until they are? Easy to add later, hard to remove. Again the `<noscript><img></noscript>` is more of a backwards-compatibility layer that we don't need to replicate here.